### ISSUE TITLE
Fix Accessibility Issue

### ISSUE THREAD

**Issue by aedwardg** [CodecademyCommunity/CodecademyCommunity.github.io#77]
## Describe the bug
We currently have an accessibility warning from ES Lint that we've temporarily disabled.  This should be fixed.
https://github.com/CodecademyCommunity/CodecademyCommunity.github.io/blob/7c5bb6d9cac5cd76c04c73e3cb613e7ba9e246e4/src/Components/Header.jsx#L20-L28

## Expected behavior
Warning should not occur and hamburger menu should look the same.

## To Reproduce
1. Remove comment shown above
2. See ES Lint warning in your editor.

## Additional context
In my own experimentation, I tried changing the `<a>` to a button but that causes a strange difference in the way the hamburger menu looks when it's activated.  If we can avoid that, it would be nice, not sure why that happens.

**Comment by Victoria-DR** [comment]
Yesterday, I also tried changing it to a button tag, but it just results in another error:

> The element button has an implicit role of button. Defining this explicitly is redundant and should be avoided

**Comment by aedwardg** [comment]
Yeah, if you change it to a button you have to delete the `role="button"` line too. That makes the new error go away but the hamburger looks kinda pixelated when you do that

**Comment by Victoria-DR** [comment]
For me, it seems like it's only pixelated when it's selected. When the focus is on another element, it returns back to a non-pixelated state.

**Comment by mikejoh12** [comment]
I tried changing the anchor-tag to a button as well and did not get pixelation with chrome but noticed it with firefox. Changing it to a div with role="button" removed the ES lint warning, seems to work and not give any pixelation issues with chrome/firefox. Thoughts on making it a div?

**Comment by Victoria-DR** [comment]
@mikejoh12 That would be a solution, though it looks like we would need to also add some event handlers to allow for the button to work when the Enter or Space keys are pressed.

At the bottom of the page you linked, it does state that best practices would dictate that the visual styling of the element be adjusted rather than using an element with the `role="button"` attribute. I would still lean towards changing the styling rather than the element type itself, but I'm still open to this solution if we're not able to find a fix using styling.
